WordPress/UITestsFoundation/Screens/ReaderScreen.swift

```diff
@@ -20,6 +20,50 @@ public class ReaderScreen: ScreenObject {
         )
     }
 
+    public func openTheLastPostInApp() {
+        getTheLastPost().tap()
+    }
+
+    public func openTheLastPostInSafari() {
+        getTheLastPost().buttons["More"].tap()
+        app.buttons["Visit"].tap()
+    }
+
+    public func getTheLastPost() -> XCUIElement {
+        let postPosition = app.cells.count - 1
+        let post = app.cells.element(boundBy: postPosition)
+
+        scrollDownUntilElementIsHittable(element: post)
+        return post
+    }
+
+    private func scrollDownUntilElementIsHittable(element: XCUIElement) {
```

Comment:
This is a nice method. Probably better than the [`scrollIntoView(within:, threshold:)`](https://github.com/wordpress-mobile/WordPress-iOS/blob/01ccb6968fc68bbb3c470ba050b26a8b127ca90f/WordPress/UITestsFoundation/XCUIElement%2BScroll.swift) we already have, because it doesn't rely on already knowing the scrollable container element.

I might steal it in the future 😄